docs/docs/getting-started/providers/credentials-tutorial.mdx

```diff
@@ -53,7 +53,140 @@ export default NextAuth({
 
   </TabItem>
   <TabItem value="sveltekit" label="SvelteKit">
-    TODO SvelteKit
+
+  ### Prerequisites
+
+  This tutorial assumes you have a SvelteKit application set up. If you don't, you can follow the [SvelteKit tutorial](https://kit.svelte.dev/docs/creating-a-project) to get started.  
+
+  ### Installing Auth.js
+
+  ```bash npm2yarn
+  npm install @auth/core @auth/sveltekit
+  ```
+
+  ### Adding environment variables
+
+  In your project root, create a `.env.local` file and add the `AUTH_SECRET` environment variable:
+
+  ```title=".env.local"
+  AUTH_SECRET="This is an example"
+  ```
+
+  `AUTH_SECRET` is a random string used by the library to encrypt tokens and email verification hashes, and **it's mandatory to keep things secure**! 🔥 🔐 . You can use:
+
+  ```
+  openssl rand -base64 32
+  ```
+
+  or https://generate-secret.vercel.app/32 to generate a random value for it.
+
+  ### Create server hook
+
+  Create the following [Server hook](https://kit.svelte.dev/docs/hooks) file. This route contains the necessary configuration for Auth.js, as well as the dynamic route handler:
+
+  ```ts title="src/hooks.server.ts"
+  import { SvelteKitAuth } from "@auth/sveltekit"
+  import Credentials from '@auth/core/providers/credentials'
+
+  export const handle = SvelteKitAuth({
+    providers: [
+      Credentials({
+        // You can specify which fields should be submitted, by adding keys to the `credentials` object.
+        // e.g. domain, username, password, 2FA token, etc.
+        credentials: {
+          username: {},
+          password: {}
+        },
+        authorize: async (credentials) => {
+          let user = null
+
+          // logic to get salted password
+          const saltedPasswordToCheck = passwordToSalt(credentials.password)
+          // logic to verify if user exists
+          user = await getUserFromDb(credentials.username, credentials.password)
```

Comment:
I know this is just a simple example thing, but to not lead people too far astray, this should be passing in the salted + hashed password here to get the user from db for example, right? So something like this:

```suggestion
          // logic to salt and hash password
          const pwHash = saltAndHashPassword(credentials.password)
          // logic to verify if user exists
          user = await getUserFromDb(credentials.username, pwHash)
```

Reply:
sure, completely agree.